compendium/DeclarativeServices/src/manager/ConfigurationNotifier.cpp

```diff
@@ -127,6 +126,14 @@ namespace cppmicroservices
                 auto iter = listenersMapHandle->find(pid);
                 if (iter != listenersMapHandle->end())
                 {
+                    // Make sure the properties does not contain any dynamic targets
+                    auto const tokenMapPtr = iter->second;
+                    for (auto const& tokenEntry : (*tokenMapPtr))
+                    {
+                        auto listener = tokenEntry.second;
+                        LogInvalidDynamicTargetInProperties(properties, listener.mgr);               
+                    }
```

Comment:
I don't see the value of logging this and I don't understand the restriction of allowing dynamic targets only for factory components. I think this is over constraining and should be removed.